### DISCUSSION TITLE
Trigger BUG in Lsp

### DISCUSSION THREAD

**Discussion by cxykhaos** [mason-org/mason.nvim#670] [Q&A]
Error executing vim.schedule lua callback: UnhandledPromiseRejection with the reason:
RPC[Error] code_name = unknown, code = 0, message = "cannot parse non-Go file file:///root/go/src/*/s*/*/*.py"

trigger bug when i open a go file and other language file
![image](https://user-images.githubusercontent.com/60059661/202185777-2688eac8-9104-4b5d-ae96-6694620270b4.png)

**Comment by cxykhaos** [comment]
> Error executing vim.schedule lua callback: UnhandledPromiseRejection with the reason: RPC[Error] code_name = unknown, code = 0, message = "cannot parse non-Go file file:///root/go/src/aquarius/s cripts/http_test/debuglogger.py"
> 
> trigger bug when i open a go file and other language file ![image](https://user-images.githubusercontent.com/60059661/202185777-2688eac8-9104-4b5d-ae96-6694620270b4.png)

I often punish this bug when I open two files in different languages

**Comment by williamboman** [comment]
Hello! This is unfortunately not the right place to ask questions regarding usage of LSP servers. Depending on what exactly the underlying issue is, I'd say it belongs in [nvim-lspconfig](https://github.com/neovim/nvim-lspconfig/) or the [`gopls`](https://github.com/golang/tools/tree/master/gopls) language server project (assuming this error comes from `gopls`). If you do file an issue in either place, make sure to provide more information about 1) how you set up the server (include code), 2) describe steps for how to reproduce the issue.
